doc/content/clocks.md

```diff
@@ -310,3 +312,11 @@ end
 one()
 one()
 ```
+
+### Case 2: post OCaml 5
```

Comment:
I'd rather have this as *case 1*, since I expect that our users will quickly switch to OCaml 5.

Reply:
Are they? I wasn't planning on shipping binary builds using OCaml 5 just yet frankly.